app/styles/ui/_ref.scss

```diff
@@ -0,0 +1,8 @@
+em.ref {
```

Comment:
Maybe just `.ref` so the React component can be switched to another element if needed?